app/models/user.rb

```diff
@@ -491,10 +491,6 @@ def self.find_by_mail(mail)
     where(['LOWER(mail) = ?', mail.to_s.downcase]).first
   end
 
-  def self.find_all_by_mails(mails)
```

Comment:
I've removed this call from the documents plugin just today. Did you check other plugins whether they call this?